update-files.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash -e
 
 mkdir -p src/bci_build/package/{postgres,nginx,grafana,mariadb}
+mkdir -p src/bci_build/package/postfix/entrypoint/ldap
```

Comment:
We started the one-liner rabbit hole, so let's die on that hill :stuck_out_tongue_winking_eye:  
```suggestion
mkdir -p src/bci_build/package/{postgres,nginx,grafana,mariadb,postfix/entrypoint/ldap}
```

Reply:
addressed in the latest commit refresh.